projects/plugins/jetpack/_inc/client/components/popover/index.jsx

```diff
@@ -79,12 +78,24 @@ class Popover extends Component {
 		bindWindowListeners();
 	}
 
+	isDOMNode( obj ) {
+		return obj && obj instanceof HTMLElement;
+	}
+
+	isRef( obj ) {
+		return obj && typeof obj === 'object' && 'current' in obj;
+	}
```

Comment:
Looks like the `obj &&` is probably redundant here. The `instanceof` and `typeof` checks will both do the right thing if it's null or false or undefined or 0.

Reply:
Thanks for mentioning that, added here: db9b9d6